plugins/vpp/ifplugin/ifplugin.go

```diff
@@ -52,6 +54,11 @@ import (
 	_ "go.ligato.io/vpp-agent/v3/plugins/vpp/ifplugin/vppcalls/vpp2001_324"
 )
 
+func init() {
+	// this adds ErrPluginDisabled to non-retryable errors
+	kvscheduler.NonRetryableErrors = append(kvscheduler.NonRetryableErrors, vppclient.ErrPluginDisabled)
```

Comment:
why this appending is here and not somewhere near to declaration of `ErrPluginDisabled` ?

Reply:
Because it would make `plugins/vpp` package depend on `kvscheduler`.